src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+@use 'variables' as v;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: v.$main-text-color;
+    margin-bottom: 4px;
+    transition: color 300ms;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: v.$second-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    &_stars {
+      display: flex;
+    }
+
+    &_count {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &_title {
+      color: v.$second-text-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &_sum {
+      &::after {
+        content: '$#{v.$price-in-string}';
+        font-weight: 700;
+        color: v.$main-text-color;
+        font-size: 16px;
+        line-height: 18px;
+        display: block;
+      }
+    }
+  }
+
+  &__button {
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    border: 1px solid v.$hover-color;
```

Comment:
Ensure that the variable 'v.$hover-color' is correctly defined in the 'variables' module. This variable is used for the button border and background color, and if it's not defined, it will cause issues.